projects/1.20-fabric/assets/wizards/wizards/lang/zh_cn.json

```
@@ -0,0 +1,83 @@
+{
+  "itemGroup.wizards.general": "魔法师",
+
+  "item.wizards.arcane_spell_book": "奥秘卷册",
+  "item.wizards.fire_spell_book": "火焰卷册",
+  "item.wizards.frost_spell_book": "寒冰卷册",
+
+  "item.wizards.wand_novice": "新手法杖",
+  "item.wizards.wand_arcane": "奥秘法杖",
+  "item.wizards.wand_fire": "火焰法杖",
+  "item.wizards.wand_frost": "寒冰法杖",
+  "item.wizards.staff_arcane": "奥秘权杖",
+  "item.wizards.staff_fire": "火焰权杖",
+  "item.wizards.staff_frost": "寒冰权杖",
+  "item.wizards.wand_netherite_arcane": "下界合金奥秘法杖",
+  "item.wizards.wand_netherite_fire": "下界合金火焰法杖",
+  "item.wizards.wand_netherite_frost": "下界合金寒冰法杖",
+  "item.wizards.staff_netherite_arcane": "下界合金奥秘权杖",
+  "item.wizards.staff_netherite_fire": "下界合金火焰权杖",
+  "item.wizards.staff_netherite_frost": "下界合金寒冰权杖",
+  "item.wizards.staff_crystal_arcane": "奥秘结晶权杖",
+  "item.wizards.staff_ruby_fire": "火焰鲁比权杖",
+  "item.wizards.staff_smaragdant_frost": "寒冰翡翠权杖",
+  "item.wizards.wizard_robe_head": "魔法师帽",
+  "item.wizards.wizard_robe_chest": "魔法师上身袍",
+  "item.wizards.wizard_robe_legs": "魔法师下身袍",
+  "item.wizards.wizard_robe_feet": "魔法师靴",
+  "item.wizards.arcane_robe_head": "奥秘魔法师帽",
+  "item.wizards.arcane_robe_chest": "奥秘魔法师上身袍",
+  "item.wizards.arcane_robe_legs": "奥秘魔法师下身袍",
+  "item.wizards.arcane_robe_feet": "奥秘魔法师靴",
+  "item.wizards.fire_robe_head": "火焰魔法师帽",
+  "item.wizards.fire_robe_chest": "火焰魔法师上身袍",
+  "item.wizards.fire_robe_legs": "火焰魔法师下身袍",
+  "item.wizards.fire_robe_feet": "火焰魔法师靴",
+  "item.wizards.frost_robe_head": "寒冰魔法师帽",
+  "item.wizards.frost_robe_chest": "寒冰魔法师上身袍",
+  "item.wizards.frost_robe_legs": "寒冰魔法师下身袍",
+  "item.wizards.frost_robe_feet": "寒冰魔法师靴",
+
+  "spell.wizards.arcane_missile.name": "奥秘飞弹",
+  "spell.wizards.arcane_missile.description": "射出能量飞弹，造成 {damage} 点奥秘法术伤害。",
+  "spell.wizards.arcane_blast.name": "奥秘爆裂",
+  "spell.wizards.arcane_blast.description": "使目标爆炸，造成 {damage} 点奥秘法术伤害。同时赋予奥秘魔能，最多可以堆叠 {effect_amplifier} 次。",
+  "spell.wizards.arcane_beam.name": "奥秘射线",
+  "spell.wizards.arcane_beam.description": "引导一束射线，每秒造成 {damage} 点奥秘法术伤害。消耗奥秘魔能。",
+
+  "spell.wizards.fireball.name": "火球术",
+  "spell.wizards.fireball.description": "射出火球，造成 {damage} 点火焰法术伤害。",
+  "spell.wizards.fire_breath.name": "火焰吐息",
+  "spell.wizards.fire_breath.description": "焚烧前方的目标，每秒造成 {damage} 点火焰法术伤害。",
+  "spell.wizards.fire_meteor.name": "陨石",
+  "spell.wizards.fire_meteor.description": "向目标坠落陨石，在 {impact_range} 格的范围内造成 {damage} 点火焰法术伤害。",
+
+  "spell.wizards.frostbolt.name": "冰球术",
+  "spell.wizards.frostbolt.description": "射出冰球，造成 {damage} 点寒冰法术伤害，同时使目标减速。",
+  "spell.wizards.frost_nova.name": "寒冰新星",
+  "spell.wizards.frost_nova.description": "使周围的目标冻结 {effect_duration} 秒，造成 {damage} 点寒冰法术伤害并阻碍其行动。处于冻结状态的敌人更容易受到寒冰法术的伤害。",

```

关于`blocking their movement`和`are vulnerable to frost magic`，提供我自己的译法
```suggestion
  "spell.wizards.frost_nova.description": "使周围的目标冻结 {effect_duration} 秒，造成 {damage} 点寒冰法术伤害并阻止其移动。处于冻结状态的敌人受到的寒冰法术伤害增加。",
```

---

已修正